src/video.c

```diff
@@ -235,6 +257,10 @@ static void video_new_window(uint display, uint w, uint h, bool fs, bool resizab
 		flags |= SDL_WINDOW_RESIZABLE;
 	}
 
+	if(video.backend == VIDEO_BACKEND_COCOA) {
+		flags |= SDL_WINDOW_ALLOW_HIGHDPI;
+	}
```

Comment:
I think we should support DPI scaling on any platform that SDL can handle. Let's remove this condition and see what breaks.